app/src/main/java/org/asteroidos/sync/connectivity/MediaService.java

```diff
@@ -293,16 +297,35 @@ public void onActiveSessionsChanged(List<MediaController> controllers) {
                 mMediaCallback.onMetadataChanged(mMediaController.getMetadata());
                 if (mMediaController.getPlaybackState() != null)
                     mMediaCallback.onPlaybackStateChanged(mMediaController.getPlaybackState());
-                Log.d("MediaService", "MediaController set: " + mMediaController.getPackageName());
+                Log.d(TAG, "MediaController set: " + mMediaController.getPackageName());
                 SharedPreferences.Editor editor = mSettings.edit();
                 editor.putString(PREFS_MEDIA_CONTROLLER_PACKAGE, mMediaController.getPackageName());
                 editor.apply();
             }
         } else {
             byte[] data = new byte[]{0};
-            mDevice.write(AsteroidUUIDS.MEDIA_ARTIST_CHAR, data, MediaService.this);
-            mDevice.write(AsteroidUUIDS.MEDIA_ALBUM_CHAR, data, MediaService.this);
-            mDevice.write(AsteroidUUIDS.MEDIA_TITLE_CHAR, data, MediaService.this);
+            mDevice.send(AsteroidUUIDS.MEDIA_ARTIST_CHAR, data, MediaService.this);
+            mDevice.send(AsteroidUUIDS.MEDIA_ALBUM_CHAR, data, MediaService.this);
+            mDevice.send(AsteroidUUIDS.MEDIA_TITLE_CHAR, data, MediaService.this);
         }
     }
+
+    @Override
+    public HashMap<UUID, Direction> getCharacteristicUUIDs() {
+
+        HashMap<UUID, Direction> chars = new HashMap<>();
+        chars.put(AsteroidUUIDS.MEDIA_TITLE_CHAR, Direction.TO_WATCH);
+        chars.put(AsteroidUUIDS.MEDIA_ALBUM_CHAR, Direction.TO_WATCH);
+        chars.put(AsteroidUUIDS.MEDIA_ARTIST_CHAR, Direction.TO_WATCH);
+        chars.put(AsteroidUUIDS.MEDIA_PLAYING_CHAR, Direction.TO_WATCH);
+        chars.put(AsteroidUUIDS.MEDIA_COMMANDS_CHAR, Direction.FROM_WATCH);
+        chars.put(AsteroidUUIDS.MEDIA_VOLUME_CHAR, Direction.TO_WATCH);
+
```

Comment:
nit: in other services you also don't have that empty line here, consistency is always better